### DISCUSSION TITLE
Issues upgrading from v2 to v3

### DISCUSSION THREAD

**Discussion by glenncarr** [JanDeDobbeleer/oh-my-posh#328] [Troubleshoot] [answered]
I apologize in advance if I've overlooked some obvious documentation.  I'm trying to upgrade from v2 to v3 and seeing the following.  How do I explicitly uninstall v2?  Using PowerShell Core 7.1.0.

![image](https://user-images.githubusercontent.com/367292/103921047-a0267e00-50d7-11eb-8224-8b88e4cb9f44.png)

Thanks,
Glenn

**Comment by lnu** [comment]
`uninstall-module oh-my-posh` then `Install-Module oh-my-posh -allowprerelease` should do the trick(I did it right now to test)
if you find any flaw in the process, don't hesitate to update the doc: https://ohmyposh.dev/docs/upgrading

you can also add the `-verbose` parameter to your command to see what's happening under the hood.

**Reply by glenncarr** [reply]
thanks, here's the error when attempting to uninstall...
![image](https://user-images.githubusercontent.com/367292/104096021-b0696500-525f-11eb-898f-d71b9e0da947.png)

**Comment by lnu** [comment, marked as answer]
You can try to remove it by hand(like [here](https://stackoverflow.com/questions/57531370/how-to-uninstall-module-of-older-version)). Was it installed using PowershellGet?
You can check installed modules with `Get-InstalledModule -Verbose`.
